streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -92,6 +107,24 @@ public void shouldFetchFhirResourceAndUploadToFhirStore() throws IOException {
 		Mockito.verify(parquetUtil, Mockito.never()).write(Mockito.<Resource> any());
 	}
 	
+	@Test
+	public void shouldGetUuidFromParent() throws SQLException, PropertyVetoException, ClassNotFoundException {
+		
+		Map<String, String> messageBody = DebeziumTestUtil.genExpectedBody();
+		Map<String, Object> messageHeaders = DebeziumTestUtil.genExpectedHeaders(Operation.CREATE, "patient");
+		String uuid = getUuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
+		
+		lenient().when(jdbcConnectionUtil.createStatement()).thenReturn(statement);
```

Comment:
Why are we using `lenient` here (and next line)? If these stubs are not needed for this test, then please remove them. If they are needed, why `lenient`?